lib/gruntTasks.js

```diff
@@ -112,6 +120,7 @@ module.exports = function (grunt) {
     sails = new Sails();
     sails.lift(sailsConfig, function (err) {
       var url;
+      var args = buildDbMigrateArgs(grunt, sails, command);
```

Comment:
Instead of passing the entire `sails` object in, please pass in `sails.config.migrations`. Building the database args shouldn't need anything more than the migrations config.